### ISSUE TITLE
Daily Counts as default for Recent Numbers

### ISSUE THREAD

**Issue by brunobowden** [WorldHealthOrganization/app#1663]
## Summary

Recent Numbers should show the "Daily" counts as the default (and left tab). The "Total" should be clickable as the right tab. The reason for this is that day to day, the "Total" graph changes very little... what's more important is the derivative, i.e. is the problem growing or shrinking? That means showing the daily stats by default. This matches the [WHO COVID-19 dashboard](http://covid19.who.int/) which also shows daily counts by default.

The main page should continue showing the total stats but also rotate through the daily stats as well in a list of four:
```
National Cases Daily
National Cases Total
Global Cases Daily
Global Cases Total
```
## Checklist:

- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

**Comment by brunobowden** [comment]
Fixed by #1673 

@Henrymarks1 - this is enough of a fix. On second thoughts, let's keep the main display as is with the total counts. Then show the daily ones in the Recent Numbers. So you can consider this all fixed up.

**Comment by Henrymarks1** [comment]
@brunobowden Yeah i agree!
